scipy/special/_add_newdocs.py

```diff
@@ -2316,18 +2316,18 @@ def add_newdoc(name, doc):
 
     Notes
     -----
-    Wrapper for the Cephes [1]_ routine `ellpj`.
+    Wrapper for the Cephes [1]_ routine ``ellpj``.
 
     These functions are periodic, with quarter-period on the real axis
-    equal to the complete elliptic integral `ellipk(m)`.
+    equal to the complete elliptic integral ``ellipk(m)``.
 
-    Relation to incomplete elliptic integral: If `u = ellipkinc(phi,m)`, then
-    `sn(u|m) = sin(phi)`, and `cn(u|m) = cos(phi)`. The `phi` is called
-    the amplitude of `u`.
+    Relation to incomplete elliptic integral: If ``u = ellipkinc(phi,m)``, then
+    ``sn(u|m) = sin(phi)``, and ``cn(u|m) = cos(phi)``. The ``phi`` is called
+    the amplitude of ``u``.
```

Comment:
```suggestion
    the amplitude of `u`.
```

Reply:
I am not sure about this, I am not sure `u` refers to the input of the function but the definition on 2324

Reply:
are you sure that `u` isn't also referring to the parameter too, though? If not, it would probably be helpful to change the name.

Reply:
Actually on a second reading I think this is ok